cypress/support/cypress-support/src/main.js

```diff
@@ -132,8 +123,36 @@ export default function (module) {
   // beforeEach
   beforeEach(() => {
     UTILS.getEnvVariable(CONSTANTS.FB_INTERACTIONLOGS).clearLogs();
-    cy.getBeforeOperationObject();
-    UTILS.destroyGlobalObjects([CONSTANTS.LIFECYCLE_APP_OBJECT_LIST]);
```

Comment:
why are we adding a .then for lines 135 and 136. 
You can create a new command which internally creates marker and save start time there. Then that command can be used in beforeeach right

Reply:
created initiatePerformanceMetrics command to save start time and create marker